src/client/sandboxed-jquery.js

```diff
@@ -1719,7 +1719,11 @@ export function init (window, undefined) {
                     container.style.zoom = 1;
                 }
 
-                body.removeChild(container);
+                // NOTE: in some cases Firefox can't remove container div,
```

Comment:
Maybe it's better to place a link to this [comment](https://github.com/DevExpress/testcafe-legacy-api/issues/10#issue-174244330) ?